application/src/main/java/org/togetherjava/tjbot/commands/tag/TagManageCommand.java

```diff
@@ -0,0 +1,231 @@
+            event.reply("You need the MESSAGE_MANAGE permission to use this command!")
+                .setEphemeral(true)
+                .queue();
+
+            return;
+        }
+
+        switch (event.getSubcommandName()) {
+            case "edittag" -> editTag(event);
+            case "editidtag" -> editIdTag(event);
+            case "deletetag" -> deleteTag(event, member);
+            case "createtag" -> createTag(event);
+            case "createidtag" -> createIdTag(event);
+            default -> throw new IllegalStateException();
+        }
+    }
+
+    private void editTag(SlashCommandEvent event) {
+        String tagId = event.getOption("id").getAsString();
+        String content = event.getOption(CONTENT).getAsString();
+
+        if (!tagSystem.exists(tagId)) {
+            event.reply(THIS_TAG_DOES_NOT_EXIST).setEphemeral(true).queue();
+
+            return;
+        }
+
+        tagSystem.put(tagId, content);
+
+        event
+            .replyEmbeds(new EmbedBuilder().setColor(Color.GREEN)
+                .setTitle(SUCCESS)
+                .setTimestamp(LocalDateTime.now())
+                .setFooter(event.getUser().getAsTag())
+                .setDescription("Successfully edited tag '" + tagId + "'")
+                .build())
+            .queue();
+    }
+
+    private void editIdTag(SlashCommandEvent event) {
+        String tagId = event.getOption(TAG_MINUS_ID).getAsString();
+        long messageId = event.getOption(MESSAGE_MINUS_ID).getAsLong();
+
+        if (!tagSystem.exists(tagId)) {
+            event.reply(THIS_TAG_DOES_NOT_EXIST).setEphemeral(true).queue();
+
+            return;
+        }
+
+        event.getMessageChannel().retrieveMessageById(messageId).queue(message -> {
+            tagSystem.put(tagId, message.getContentRaw());
+
+            event
+                .replyEmbeds(new EmbedBuilder().setColor(Color.GREEN)
+                    .setTitle(SUCCESS)
+                    .setTimestamp(LocalDateTime.now())
+                    .setFooter(event.getUser().getAsTag())
+                    .setDescription("Successfully edited tag '" + tagId + "'")
+                    .build())
+                .queue();
+        }, failure -> {
+            if (failure instanceof ErrorResponseException ex
+                    && ex.getErrorResponse() == ErrorResponse.UNKNOWN_MESSAGE) {
+                event.reply("This message doesn't exist").setEphemeral(true).queue();
+            }
+        });
+    }
+
+    private void deleteTag(SlashCommandEvent event, Member member) {
+        String tagId = event.getOption("id").getAsString();
+
+        if (!tagSystem.exists(tagId)) {
+            event.reply(THIS_TAG_DOES_NOT_EXIST).setEphemeral(true).queue();
+
+            return;
+        }
+
+        event.reply("You sure? Confirming this will delete the tag '" + tagId + "'")
+            .addActionRow(
+                    Button.of(ButtonStyle.SUCCESS, generateComponentId(member.getId(), tagId),
```

Comment:
Switch them around. Deleting is the destructive button and should be red, aborting is the safe option and should be green/neutral.